lib/dialects/abstract/query-generator.js

```diff
@@ -970,7 +999,7 @@ class QueryGenerator {
    *
    * @returns {string}
    */
-  quoteTable(param, alias) {
+  quoteTable(param, alias = false) {
```

Comment:
I think it was because during debug, I had some IDE warnings regarding this method being called with only one parameter. So I added `= false` to be explicit about the fact that it's optional

Not really related to the point of this PR though